src/pages/extensions/iter8/Iter8ExperimentDetails/ExperimentCreatePage.tsx

```diff
@@ -546,22 +561,27 @@ class ExperimentCreatePage extends React.Component<Props, State> {
                 <GridItem span={6}>
                   <FormGroup
                     fieldId="candidate"
-                    label="Candidate"
+                    label="Select Candidate"
                     isRequired={true}
                     isValid={this.state.experiment.candidate !== ''}
                     helperText="The candidate deployment of the target service (i.e. reviews-v2)"
                     helperTextInvalid="Candidate deployment cannot be empty"
                   >
-                    <FormSelect
+                    <TextInput
```

Comment:
Is this on purpose ?
It changes a little bit the both components:
![image](https://user-images.githubusercontent.com/1662329/83006768-fc3e0580-a012-11ea-82b9-f97360fa5cc0.png)
Baseline is fixed to workloads but Candidate can enter any text.

I guess both should be use same model, if free format is included, I'd also add a validation on enter.

Reply:
Note, that a side effect of this is:
![image](https://user-images.githubusercontent.com/1662329/83006997-4f17bd00-a013-11ea-81e0-cbac8451ae8d.png)

When the candidate is populated, then it filters the list of possible workloads, I thought it was a bug but I saw the change of the component.

To see all workloads, user may delete some text then (I've tested with the reviews service).

Reply:
Yes, the candidate is intended to be selectable from the existing workload version, or user can type-in non-existing one. But baseline does not make sense to have non-existing workload.

As for user need to delete the pre-filtered one to show the whole list,  I will change the behavior so that the field will be without default ( instead default to the first one in the list). 

what do you think?